solution/zenko-version-cr.yaml

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: zenko.io/v1alpha1
+kind: ZenkoVersion
+metadata:
+  name: 1.1-dev
```

Comment:
shouldn't it be `2.0.0-dev`?

Reply:
ooh, I can use the git tag here as well so we don't have to update it for each release. cc @rahulreddy @RachedBenMustapha